src/explorer/CosmosDBTreeItem.ts

```diff
@@ -202,44 +290,53 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
         return result;
     }
 
-    private async promptForMongoAppSettings(appSettingsDict: StringDictionary, database: DatabaseTreeItem): Promise<Map<string, string>> {
+    private async promptForAppSettings(appSettingsDict: StringDictionary, accountAppSettings: Map<string | undefined, string | undefined>, suffixes: string[] | undefined, errorMsg: string, defaultPrefixString: string): Promise<Map<string, string>> {
         const prompt: string = 'Enter new connection setting key';
```

Comment:
This message needs to be adjusted based on whether or not there's suffixes, but it feels like overkill to pass in the messages. How about you do this:
```typescript
const prompt: string = suffixes ? localize('enterPrefix', 'Enter new connection setting prefix') : localize('enterKey', 'Enter new connection setting key');
```

And do something similar for the error message below, instead of passing it in.